src/Features/CSharpTest/ConvertPrimaryToRegularConstructor/ConvertPrimaryToRegularConstructorTests.cs

```diff
@@ -2467,14 +2468,13 @@ class [|C(int i)|]
             FixedCode = """
                 class C
                 {
-                    public C(int i)
-                    {
-                    }
 
                     #region constructors
 
                     #endregion
-
+                    public C(int i)
```

Comment:
What is it about calling `EnsureOpenAndCloseBraceTokens` that caused this constructor to change where it was generated?

Reply:
Never mind. I should have just looked.
https://github.com/dotnet/roslyn/blob/151e2d7e229ee421834f22b28ee8abb10161ed2a/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/TypeDeclarationSyntaxExtensions.cs#L121-L124